acceptance-tests/src/test/resources/uk/gov/di/test/features/Password_reset_forced.feature

```diff
@@ -0,0 +1,197 @@
+@PasswordResetForced
+Feature: User forced to reset their One Login password
+
+  Scenario: Sms app user has to reset their password when password reset intervention is placed on their account (happy path)
+#  Scenario : User resets their password from the intervention ‘You need to reset your password’ screen
+    Given TICF then "reset password" user account: "TEST_USER_ACCOUNT_RECOVERY_EMAIL_1"
+    And the user comes from the stub relying party with options: "default"
+    Then the user is taken to the "Create a GOV.UK One Login or sign in" page
+    When the user selects sign in
+    Then the user is taken to the "Enter your email" page
+    When user enters "TEST_USER_PASSWORD_RESET_ACCOUNT_EMAIL_1" email address
+    Then the user is taken to the "Enter your password" page
+    When the user enters their password
+#Start of new functionality
+    Then the user is taken to the "Check your phone" page
+    When the user enters the six digit security code from their phone
+    Then the user is taken to the "You need to reset your password" page
+    When user selects continue
+    Then the user is taken to the "Check your email" page
+    When the user enters the six digit security code from their email
+    Then the user is taken to the "Reset your password" page
+    When the user enters valid new password and correctly retypes it
+#End of new functionality
+    Then the user is taken to the "Example - GOV.UK - User Info" page
+
```

Comment:
Need to add the logout steps otherwise it messes with the users session for other tests:
When the user clicks logout
Then the user is taken to the "Signed out" page